src/clj/game/cards/resources.clj

```diff
@@ -1316,6 +1316,7 @@
              {:effect (effect (update! (assoc card :slums-active true)))}
              :pre-trash
              {:req (req (and (:slums-active card)
+                             (not (= (:title target) "Mumbad Virtual Tour"))
```

Comment:
I'd prefer an implementation that doesn't couple this card to the specific name of another card. Let's go with a `:flag` in MVT and then Slums can check the target's card-def for that flag.